### PR TITLE
Api call

### PR DESCRIPTION
**Pull request by tsohleDev**
Make API calls to [Leaderboard API documentation](https://www.notion.so/microverse/Leaderboard-API-service-24c0c3c116974ac49488d4eb0267ade3) and integrate the data with the app

### CODE REVIEW

File: src/modules/list.node.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    });
+
+    this.loader.style.display = 'none';
+    this.listItems.forEach((element) => { element.style.display = 'block'; });
+  }
+
+    #addScore(name, score) {
+    const li = document.createElement('li');
+    li.innerHTML = `${name}:  ${score}`;
+    li.setAttribute('id', 'li');
```
Comment (m05e5) [inline src/modules/list.node.js:48]: - [x] When trying to add a score, the name and score from the input fields are not being passed in the post request. Kindly fix it
-
image

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @tsohleDev , 

Good job so far. !
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [ ] Please check the comments below 

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/list.node.js (hunk 1 above)
Comment (tsohleDev) [reply src/modules/list.node.js:48]: Thank you, I have fixed it

**Review by OLIPLICHE — APPROVED** [review]
Hi @tsohleDev , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights
- App working fine 👍

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**